extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php

```diff
@@ -1,21 +1,56 @@
 <?php
 
-class PortableInfoboxRenderServiceTest extends PHPUnit_Framework_TestCase {
+class PortableInfoboxRenderServiceTest extends WikiaBaseTest {
 	private $infoboxRenderService;
 
 	protected function setUp() {
 		$this->setupFile = dirname( __FILE__ ) . '/../PortableInfobox.setup.php';
 		parent::setUp();
+	}
+
+	/**
+	 * @param $isWikiaMobile
+	 * @param $input to check presence of 'invalidImage' field
+	 * @return PHPUnit_Framework_MockObject_MockObject
+     */
+	private function getInfoboxRenderServiceMock( $isWikiaMobile, $input )
```

Comment:
Per offline discussion, let's use a consistent type of configuring the mock - two bools, an associative config array, or an interface similar to one in https://github.com/Wikia/app/blob/DAT-2983/extensions/wikia/ImageServing/tests/ImageServingDriverMainNSTest.php#L49-L55